src/compound/InterestRatesManager.sol

```diff
@@ -54,41 +52,40 @@ contract InterestRatesManager is IInterestRatesManager, MorphoStorage {
     function updateP2PIndexes(address _poolToken) external {
         Types.LastPoolIndexes storage poolIndexes = lastPoolIndexes[_poolToken];
```

Comment:
Misleading, as we use `poolSupplyIndex` and `poolBorrowIndex` for the current ones
```suggestion
        Types.LastPoolIndexes storage lastPoolIndexes = lastPoolIndexes[_poolToken];
```